arches/app/models/models.py

```diff
@@ -172,19 +175,21 @@ class Meta:
 
 
 class FormXCard(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid1) 
     form = models.ForeignKey(Form, db_column='formid')
-    parentcard = models.ForeignKey(Card, db_column='parentcardid')
+    card = models.ForeignKey(Card, db_column='cardid')
 
     class Meta:
         managed = True
         db_table = 'forms_x_card'
-        unique_together = (('form', 'parentcard'),)
+        unique_together = (('form', 'card'),)
 
 
 class NodeGroup(models.Model):
     nodegroupid = models.UUIDField(primary_key=True, default=uuid.uuid1)  # This field type is a guess.
-    cardinality = models.TextField()
+    cardinality = models.TextField(blank=True, null=True)
```

Comment:
I think every nodegroup needs to specify a cardinality, maybe we can specify a default value as well